tests/components/vodafone_station/test_device_tracker.py

```diff
@@ -0,0 +1,51 @@
+"""Define tests for the Vodafone Station device tracker."""
+
+from unittest.mock import AsyncMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.vodafone_station.const import DOMAIN, SCAN_INTERVAL
+from homeassistant.components.vodafone_station.coordinator import CONSIDER_HOME_SECONDS
+from homeassistant.core import HomeAssistant
+
+from .const import DEVICE_1, DEVICE_1_MAC, DEVICE_DATA_QUERY, MOCK_USER_DATA
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_coordinator_consider_home(
+    hass: HomeAssistant,
+    freezer: FrozenDateTimeFactory,
+    mock_vodafone_station_router: AsyncMock,
+) -> None:
+    """Test if device is considered not_home when disconnected."""
+
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_DATA)
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.vodafone_station.device_tracker.VodafoneStationTracker.entity_registry_enabled_default",
+        return_value=True,
+    ):
```

Comment:
there's a built in fixture for that `@pytest.mark.usefixtures("entity_registry_enabled_by_default")`